app/lib/features/news/providers/notifiers/news_list_notifier.dart

```diff
@@ -43,13 +43,24 @@ class AsyncNewsListNotifier
   Future<List<NewsEntry>> _fetchNews(Client client) async {
     //GET ALL NEWS
     if (arg == null) {
-      return (await client.latestNewsEntries(25))
-          .toList(); // this might throw internally
+      return sortNewsListDscTime(
+        (await client.latestNewsEntries(25)).toList(),
+      ); // this might throw internally
     } else {
       //GET SPACE NEWS
       final space = await client.space(arg!);
-      return (await space.latestNewsEntries(100))
-          .toList(); // this might throw internally
+      return sortNewsListDscTime(
+        (await space.latestNewsEntries(100)).toList(),
+      ); // this might throw internally
     }
   }
 }
+
+Future<List<NewsEntry>> sortNewsListDscTime(
+  List<NewsEntry> newsList,
+) async {
+  newsList.sort(
+    (a, b) => b.originServerTs().compareTo(a.originServerTs()),
+  );
+  return newsList;
+}
```

Comment:
I am confused ... the `latest` doesn't give them to you in the right order?

Reply:
I was also bit surprised. There were not any sorting issue in my testing but @emilvincentz reported this so that added flutter side sorting.